### PR TITLE
Changed instruction not to include Eclipse /GPE versions

### PR DESCRIPTION
**Pull request by joelhandwell**

### CODE REVIEW

File: src/main/markdown/usingeclipse.md [hunk 1]
```diff
@@ -15,21 +15,21 @@ GWT even easier.
...
-If you do not already have Eclipse, you may download it from the [Eclipse Website](http://www.eclipse.org/downloads/). We suggest downloading Eclipse 4.3 (Kepler).
+If you do not already have Eclipse, you may download it from the [Eclipse Website](http://www.eclipse.org/downloads/). We suggest downloading Eclipse 4.4 (Luna).
 
 ## Install the Plugin<a id="installing"></a>
 
 <img src="https://developers.google.com/eclipse/images/google-plugin.png" style="float: left; width: 40px;
 margin-right: 15px;"/>
 
-Install the Google Plugin for Eclipse 4.3 by using the following update site:
+Install the Google Plugin for Eclipse 4.4 by using the following update site:
```
Comment (jDramaix) [inline src/main/markdown/usingeclipse.md]: I'm wondering if we shouldn't refer directly https://developers.google.com/eclipse/docs/download
@dankurka

Comment (dankurka) [inline src/main/markdown/usingeclipse.md]: I think that would be better, this way we do not have to keep track of all GPE releases

Comment (branflake2267) [inline src/main/markdown/usingeclipse.md]: I suspect the version could be removed all together.

**Comment by joelhandwell** [comment]
@jDramaix @dankurka @branflake2267 Updated not to specify Eclipse / GPE version numbers.

File: src/main/markdown/usingeclipse.md [hunk 2]
```diff
@@ -15,21 +15,16 @@ GWT even easier.
...
-Install the Google Plugin for Eclipse 4.3 by using the following update site:
+Install the Google Plugin for Eclipse following instruction on this page:
 
-`http://dl.google.com/eclipse/plugin/4.3`
-
-If you are using an earlier version of Eclipse, replace the 4.3 version number
-with your version (4.2 or 3.7). For detailed instructions on installing plugins
-in Eclipse, see instructions for [Eclipse 4.3](//developers.google.com/eclipse/docs/install-eclipse-4.3),
-[Eclipse 3.8/4.2](https://developers.google.com/eclipse/docs/install-eclipse-4.2)
+`https://developers.google.com/eclipse/docs/download`
```
Comment (jDramaix) [inline src/main/markdown/usingeclipse.md]: Use a link to make it clickable:

Install the Google Plugin for Eclipse following [these instructions](https://developers.google.com/eclipse/docs/download)

**Comment by joelhandwell** [comment]
@jDramaix thanks, updated to use link.

**Comment by jDramaix** [comment]
LGTM

**Pull request merged**